nifi-commons/nifi-utils/src/main/java/org/apache/nifi/stream/io/util/TextLineDemarcator.java

```diff
@@ -270,5 +180,10 @@ public boolean isStartsWithMatch() {
         void setStartsWithMatch(int startsWithMatch) {
```

Comment:
Why does this take in an `int` and then assign a `boolean` based on it? Why not just take in an `boolean`?